master/internal/api_experiment.go

```diff
@@ -1565,12 +1601,19 @@ func (a *apiServer) ContinueExperiment(
 		// TODO consider moving this to trial_id_task_id or some other level to preserve
 		// the history of what happened during the trial. We should also do this
 		// with submitted config yamls likely and display these in the webui.
-		if _, err := tx.NewUpdate().Model(&model.Trial{}).
-			Set("restarts = 0").
-			Set("end_time = null").
-			Where("id = ?", trialsResp.Trials[0].Id).
```

Comment:
can we do something like this?

```
var trialIDs []int
for _, t := range trialsResp.Trials {
...
}

if len(trialIDs) > 0 {
...
.Where("id IN (?)", bun.In(trialIDs))
...
}
```

I the ```tx.NewValues``` is a little confusing to me, I think it also can lead to very long query logs if the query fails for some reason

Reply:
done